packages/account/src/providers/provider.test.ts

```diff
@@ -1188,50 +1183,6 @@ Supported fuel-core version: ${mock.supportedVersion}.`
     );
   });
 
-  it('should ensure fuel node version warning is shown before chain incompatibility error', async () => {
```

Comment:
This was added in #2925. My argument is that these chain incompatibility errors won't be happening anymore due to great care taken by the `fuel-core` team in ensuring backwards compatibility, so we might as well favor optimizing the query over showing the warning.

Reply:
Do we need to make a network request to make this check?

Reply:
The flow was:
1. fetch node info
2. if fuels doesn't support the node, write a warning to the user
3. fetch the chain info
4. if the chain info query fails, the user will at least have the knowledge that the sdk doesn't support the `fuel-core` version they're querying

The feature was motivated by the frequent breaking changes happening to the graphql api, but it's stable now and these errors shouldn't occur, so IMO we should favor performance.

Reply:
> I'll have a think on this one - although I agree with the sentiment and there has been a lot of confusion around this warning - maybe removal at this time is not right.

The warning won't be there only if this specific query `getChainAndNodeInfo` fails because it's logged right after the query is successful. If this query is successful, the warning will be logged and can be of help if some subsequent queries fail.

Reply:
Fair. I think the benefits outweigh the cons of removing this feature here.